tools/drdeploy.c

```diff
@@ -1737,6 +1785,9 @@ _tmain(int argc, TCHAR *targv[])
     if (limit == 0 && !use_ptrace && !kill_group) {
         info("will exec %s", app_name);
         errcode = dr_inject_prepare_to_exec(app_name, app_argv, &inject_data);
+    } else if (attach_pid != 0) {
+        errcode =
+            dr_inject_prepare_to_attach(attach_pid, app_name, wait_syscall, &inject_data);
```

Comment:
A regression test needs to be added.  Separating it into a separate PR would be ok so long as it's forthcoming.

Reply:
I'll add in another PR